docs/api/power-save-blocker.md

```diff
@@ -0,0 +1,49 @@
+# power-save-blocker
+
+The `power-save-blocker` module is used to block the system from entering
+low-power(sleep) mode, allowing app to keep system and screen active.
+
+An example is:
+
+```javascript
+var powerSaveBlocker = require('power-save-blocker');
+
+var id = powerSaveBlocker.start(powerSaveBlocker.PREVENT_DISPLAY_SLEEP);
```

Comment:
I think we should use String as parameter name, like the clipboard and dialog module:

```js
powerSaveBlocker.start('prevent-display-sleep');
```

It is very rare to use constant integers as parameters in JavaScript.